src/app/collections/PresentationNode.tsx

```diff
@@ -228,7 +229,7 @@ class PresentationNode extends React.Component<Props> {
             </div>
           </div>
         )}
-        {childrenExpanded && presentationNodeHash === 1024788583 && (
+        {childrenExpanded && presentationNodeHash === TRIUMPHS_ROOT_NODE && (
```

Comment:
Believe it or not this hash comes down in the profile response now... I have a branch where I tried to wire it up but something or other made it difficult...

Reply:
oh nice wow
![image](https://user-images.githubusercontent.com/68782081/88992596-ae56af80-d298-11ea-845b-91d6110eac93.png)